printer_mods/chirpy/RunoutUnklicky/runoutunklicky.cfg

```diff
@@ -0,0 +1,28 @@
+# Uncomment and set the correct value for the switch_pin below
+
+[pause_resume]
+
+[gcode_macro M600]
+gcode:
+    {% set X = params.X|default(50)|float %}
```

Comment:
While the rest of the mod looks fine to me, this has me a little worried. That's a fine way to crash the printer as that stops at random places depending on what's printing. I would love to see some safety calculations done here to prevent crashes.
IE: use printer.toolhead.axis_maximum.x or minimum and printer.toolhead.position.x to calculate a safe move. I think fluidd/mainsail's pause macro has a great example of this for learning purposes.

Reply:
That's fair. I've updated the sample configuration to provide the minimum needed and linked to two examples of how to incorporate comprehensive PAUSE/RESUME macros in klipper. This seems better than just copying one of those into this mod which is primarily for the hardware creation of the runout sensor.